resource/logging_macros.h.em

```diff
@@ -374,7 +231,7 @@ def get_macro_arguments(suffix):
 // empty logging macros for severity @(severity) when being disabled at compile time
 @[ for suffix in feature_combinations]@
 /// Empty logging macro due to the preprocessor definition of RCUTILS_LOG_MIN_SEVERITY.
-# define RCUTILS_LOG_@(severity)@(suffix)(@(get_macro_parameters(suffix))format, ...)
+# define RCUTILS_LOG_@(severity)@(suffix)(@(''.join([p + ', ' for p in get_macro_parameters(suffix).keys()]))format, ...)
```

Comment:
Why move this logic from the function (where it was only written once) here as well as below where it is now duplicated?

Reply:
because the way the data is being formatted is very specific to this template file (I might still use the logic in `get_macro_arguments` in rclpy, for example, but I need to format the data in a different way). there could be a function local to this file that formats the string but I didn't think it was worthwhile